semgrep_output_v1.atd

```diff
@@ -1797,12 +1799,44 @@ type output_format = [
   | Emacs
 ]
 
+type manifest = {
```

Comment:
I updated this type so that it is now essentially identical to both `manifest_target` in that file and to `manifest` in `Target.ml`. The type is still duplicated here because I wasn't aware of a better way to structure it, though. But hopefully this interface type will only be around for a few months until we get SSC into ocaml entirely

Anyway, curious to hear what you think of this